build_metal.sh

```diff
@@ -94,7 +94,7 @@ else
 fi
 
 echo "Building tt-metal"
-cmake_args="-B build -G Ninja -DCMAKE_CXX_COMPILER=clang++-17 -DCMAKE_EXPORT_COMPILE_COMMANDS=$export_compile_commands"
+cmake_args="-B build -G Ninja -DCMAKE_CXX_COMPILER=g++ -DCMAKE_EXPORT_COMPILE_COMMANDS=$export_compile_commands"
```

Comment:
Temporary

Reply:
Can't get clang based build to work with lldb or dbg debugger 😢

Reply:
Sounds good, pls delete once you're done

Or you can also add the option for compiler for us :)